crates/evm/src/execute.rs

```diff
@@ -60,56 +63,91 @@ pub struct EthBlockOutput<T> {
     pub gas_used: u64,
 }
 
+/// The output of a batch of ethereum blocks.
+#[derive(Debug)]
+pub struct BatchBlockExecutionOutput {
+    /// Bundle state with reverts.
+    pub bundle: BundleState,
+    /// The collection of receipts.
+    /// Outer vector stores receipts for each block sequentially.
+    /// The inner vector stores receipts ordered by transaction number.
+    ///
+    /// If receipt is None it means it is pruned.
+    pub receipts: Receipts,
```

Comment:
Should this be `Receipts<T>(Vec<T>)` such that it's `BatchBlockExecutionOutput<T>` like above?